scout: Update instructions for the Scout ACR integration (#18972)
docker/docs: content/scout/integrations/registry/acr.md

   Docker Scout Dashboard.
2. Select the **Analyze my images from another registry** filter option.
3. Find **Azure Container Registry** in the list, and select **Integrate**.
4. In the **How to integrate** section, enter the **Registry hostname** of the
   registry you want to integrate.
5. Select **Next**.
6. Select **Deploy to Azure** to open the template deployment wizard in Azure.

   You may be prompted to sign in to your Azure account if you're not already
   signed in.

7. In the template wizard, configure your deployment:

   - **Resource group**: enter the same resource group as you're using for the
     container registry. The Docker Scout resources must be deployed to the
     same resource group as the registry.

>    - **Registry name**: the filed will be prefilled with the subdomain of the

Reviewer: ```suggestion
   - **Registry name**: the field is pre-filled with the subdomain of the
```
